apps/system/utils/netcmd.c

```diff
@@ -89,14 +89,6 @@ extern int iotivity_cloud_main(int argc, char *argv[]);
 #ifdef CONFIG_SYSTEM_NETDB
 extern int netdb_main(int argc, char *argv[]);
 #endif
-#ifdef CONFIG_EXTERNAL_WAKAAMA_EXAMPLE
-#ifdef CONFIG_LWM2M_CLIENT_MODE
-extern int lwm2m_client_main(int argc, char *argv[]);
-#endif
-#ifdef CONFIG_LWM2M_SERVER_MODE
-extern int lwm2m_server_main(int argc, char *argv[]);
-#endif
-#endif
```

Comment:
Dear. dr-venkman

Sorry for late response.
We're preparing next pull request including separation of example for DS SW R&D center
all wakaama codes will be reverted to remove DS SW R&D center changes

Thanks